pages/tournaments/[tournamentId]/users/[id].tsx

```diff
@@ -256,6 +256,7 @@ export default function User({
             tournament={tournament}
             imageUrl={imageUrl}
             currentUserIsUmpire={currentUserIsUmpire}
+            currentUserId={user.id}
```

Comment:
Eiks tähän pitäis passata currentUserin id? Nythän tähän menee pelaajasivun käyttäjän id, eli tuon jo ylempänä annettavan `user`-objektin id, jolloin `showAlias`-tarkistuksen `user.id === currentUserId` -vertailu on aina tosi

Reply:
Varmistan kommentoimalla: koskeeko sama pointti myös koodiriviä 268?

Reply:
No joo kyllähän siihen pitäis

Reply:
Joo sama toki molempien näkymien kohdalla.